src/Microsoft.AspNetCore.DataProtection.Redis/baseline.netcore.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
We have not released a major version of this package; highest was `0.1.2`. So, don't add a baseline and instead disable API check in the .csproj file.